src/index.tsx

```diff
@@ -3,14 +3,15 @@ import ReactDOM from "react-dom/client";
 import "./index.css";
 import reportWebVitals from "./reportWebVitals";
 import { DashboardPage } from "./modules/Dashboard/pages";
+import { WelcomePage } from "./modules/Welcome/pages/WelcomePage";
 
 const root = ReactDOM.createRoot(
   document.getElementById("root") as HTMLElement
 );
 
 root.render(
   <React.StrictMode>
-    <DashboardPage />
+    <WelcomePage />
```

Comment:
Code below gives us a possibility to add more pages on different routes.
```suggestion
  <Routes>
        <Route path="/" element={<WelcomePage />} />
    <Routes />
```